src/plugins/mf_priority.cpp

```diff
@@ -1081,6 +1160,18 @@ static int inactive_cb (flux_plugin_t *p,
         b->held_jobs.erase (b->held_jobs.begin ());
     }
 
+    // fetch the time the job was released
+    double *timestamp = static_cast<double *> (flux_jobtap_job_aux_get (
+                                                    p,
+                                                    FLUX_JOBTAP_CURRENT_JOB,
+                                                    "mf_priority:t_released"));
+    if (timestamp) {
+        // remove the entry from the map
+        auto it = released_jobs.find (FLUX_JOBTAP_CURRENT_JOB);
```

Comment:
Again, it may not have been clear but `FLUX_JOBTAP_CURRENT_JOB` is a constant. To get the actual jobid you'll have to unpack it from `flux_plugin_arg_t *args`